packages/twenty-server/src/engine/api/graphql/workspace-resolver-builder/factories/create-many-resolver.factory.ts

```diff
@@ -40,7 +40,7 @@ export class CreateManyResolverFactory
 
         return await this.graphqlQueryRunnerService.createMany(args, options);
       } catch (error) {
-        workspaceQueryRunnerGraphqlApiExceptionHandler(error);
+        workspaceQueryRunnerGraphqlApiExceptionHandler(error, context);
```

Comment:
should we use internalContext here as well to keep it consistent and avoid issues with _context?